src/commands/AudacityCommand.h

```diff
@@ -120,6 +120,9 @@ class AUDACITY_DLL_API AudacityCommand /* not final */ : public wxEvtHandler,
    //! Parameters, if defined.  false means no defined parameters.
```

Comment:
Are these comments outdated now? if you will rewrite them, please could you write in which case the Const visitor version is used, vs. the non-const version. 
My guess is, the non-const version is used when we want the visitor to read and write the settings, while the other is to be used when the visitor only needs to read them. An example use-case in the comments would be helpful :-)

Reply:
You rightly noticed that the ConstSettingsVisitor class is defined but not yet used in this pull request.

It will become the base class for ShuttleGetDefinition and ShuttleGetAutomation, and the NullShuttle in EffectManager.cpp will be of that class.  Those are the visitors that do not modify the effect or command objects.

Other work must happen before I do that.  Effects, as well as commands, must have two overloads of VisitSettings so they can also accept ConstSettingsVisitor.  This PR accomplishes that only for commands.  A later PR will do it for effects.

Reply:
> Are these comments outdated now? (because they reference "Parameters") in any case I do not find them much helpful so I'd ask you to correct and expand them a bit.
> 
> Please could you also comment in which case the Const visitor version is used, vs. the non-const version. An example use-case would be helpful.

Parameters, settings... interchangeable words I think.

Basically, there is just a success code.  The base class AudacityCommand doesn't know how to visit any settings.